tools/validators/instance_validator/tests/handler_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests tools.validators.instance_validator.instance_validator"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl.testing import absltest
+from validate import handler
+
+_TESTCASE_PATH = os.path.join('.', 'tests', 'fake_instances')
+
+class HandlerTest(absltest.TestCase):
+
+  def testOneBuildingExist(self):
```

Comment:
Could you add the name of the methods under test to these test names?

Reply:
done